src/main/java/frc/robot/commands/drive/PathFollowCommand.java

```diff
@@ -33,7 +33,6 @@ public PathFollowCommand(DriveBase driveBase, PathPlannerTrajectory traj) {
         new PIDController(PathConstants.KP, PathConstants.KI, PathConstants.KD),
         new PIDController(PathConstants.KP, PathConstants.KI, PathConstants.KD),
         driveBase::tankDriveVoltage,
-        true
```

Comment:
This line was to enable `useAllianceColor`, PathPlanner's automatic path mirroring. Looks like it accidentally got removed.